.github/workflows/precommit.yml

```diff
@@ -458,7 +458,23 @@ jobs:
       matrix:
         os: [ ubuntu-latest ]
         python-version: [ "3.9", "3.11", "3.12" ]
-        cloud-provider: [ aws ]
+        cloud-provider: [aws, gcp, azure]
+        exclude:
```

Comment:
No need to change in this PR. But curious if there is a syntax to include the combinations instead of excluding?